test/mongoose_riak_SUITE.erl

```diff
@@ -0,0 +1,49 @@
+-module(mongoose_riak_SUITE).
```

Comment:
On travis, `small tests` doesn't always have a DB running, it'd be better to move these test case to `big_tests` see `big_tests/tests/mongoose_elasticsearch_SUITE` for an inspiration.

Reply:
:+1: